src/content/guides/using-persistent-volumes-on-aws-with-efs/index.md

```diff
@@ -0,0 +1,126 @@
+- Select the Availability Zone with the subnets where your instances are located and the security-groups of the workers.
+- Choose the throughput and performance mode, no file system policy or access points are needed.
+- Create the instance and note down the EFS instance id.
+
+## EFS Provisioner configuration file
+Create a file on your local machine named "efs-provisioner.yaml" with the following content:
+```yaml
+global:
+  deployEnv: production
+
+efsProvisioner:
+  efsFileSystemId: fs-90f935c8
+  awsRegion: eu-central-1
+  path: /
+  provisionerName: giantswarm.io/aws-efs
+  storageClass:
+    name: efs
+    isDefault: false
+    gidAllocate:
+      enabled: true
+      gidMin: 40000
+      gidMax: 50000
+    reclaimPolicy: Delete
+    mountOptions: []
+
+rbac:
+  create: true
+
+podSecurityPolicy:
+  enabled: true
+```
+You will need to adjust the **efsFileSystemId** and **awsRegion** to match the configured values of the EFS instance.
+
+For additional configuration parameters see [documentation](https://github.com/kubernetes-incubator/external-storage/tree/master/aws/efs).
+
+## Installing EFS Provisioner
+
+To install the provisioner you will need to follow these steps:
+- Access Happa and select the cluster you want to install the provisioner.
+- Open the "Helm Stable" catalog
+- Write "efs-provisioner" in the search bar.
+- Select the "efs-provisioner" application and then click the Configure & Install button.
+- Upload the efs-provisioner.yaml file created in the previous step.
+- Finally, you can click the Install App button and it will be installed into your cluster.
+
+
+## Using EFS Volumes
+Your Kubernetes cluster will have a new Storage Class `efs` deployed, which you will have to reference when creating persistent volumes.
+
+In order to check that the storage class exits, you should see something similar to:
+```nohighlight
+$ kubectl get storageclass
+NAME            PROVISIONER             AGE
+efs             giantswarm.io/aws-efs   23m
+gp2 (default)   kubernetes.io/aws-ebs   5h31m
+```
+
+In the following example you can see the annotation `volume.beta.kubernetes.io/storage-class` matches the "efs" Storage Class:
+```yaml
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: test
+  annotations:
+    volume.beta.kubernetes.io/storage-class: "efs"
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 100Mi
+```
+
+## Testing the new storage class
+
+You can create a file with the example above in a file called pvc_claim.yaml and instantiate with the following command:
+```nohighlight
+$ kubectl apply -f pvc_claim.yaml
+```
+In order to check the status of the volume we can check the status of the PVC:
+```nohighlight
+$ kubectl describe pvc test
+NAME   STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+test   Bound    pvc-0d7b988e-4eed-4cf7-918b-30964774fa13   100Mi        RWX            efs            5s
+```
```

Comment:
Is this really the output for the `describe` command? It looks more like a `get`.

Reply:
true!